packages/web-components/@sfx/autocomplete/stories/index.ts

```diff
@@ -22,33 +22,30 @@ const autocompleteNotesIntro = `
 <sfx-autocomplete></sfx-autocomplete>
 \`\`\`
 
-## Demonstrated in this story`
+## Demonstrated in this story`;
 
 function getAutocompleteComponent(results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = []): string {
   const optionalTitle = text('Optional Title', 'Autocomplete Results');
 
   if (results.length > 0) {
-    const autocompleteResults = text('Autocomplete Results', JSON.stringify(results))
+    const autocompleteResultsKnob = text('Autocomplete Results', JSON.stringify(results));
```

Comment:
I recently named a stub 'exmapleSomethingStub' and an interface 'saytRequestType' and I was told not to name things in that way. Do you think the same would apply to this, should we actually not have the variable end if 'stub'?

Reply:
In general, I would avoid repeating the type if it's clear from the context and there are no other conflicts. In the case of a stub, it's clear from the context that the variable holds a stub, both from its declaration and its usage. In that case, adding "stub" to the end doesn't add anything meaningful.

If the same applies in this case, then I would drop the "knob". I probably would leave it off in this case, especially since we're not making use of any knob-specific behaviour when referencing the variable.

Reply:
If that's the case, I'm okay with leaving "knob" there. Perhaps `customAutocompleteResults` if you want to remove "knob".